.gitignore

```diff
@@ -37,3 +37,5 @@ node_modules
 # Output of the go coverage tool
 *.out
 example_cluster_aws
+mongodbatlasv2/api
+mongodbatlasv2/.openapi-generator
```

Comment:
Is there any reason why we go for `mongodbatlasv2` instead of `mongodbatlas`? Isn't the release version tag enough to differentiate v1 from v2? I would expect customers to import the SDK with `go.mongodb.org/atlas v2.0.0 `, they should not need to know that this is "v2" of our API, they just want to use our latest APIs

Reply:
mongodbatlas exist and it is going to be supported for months (if not years) with mongodbatlasv2 being completely different project (not just new version with couple breaking changes - it is all new - new api, clients etc.)
Lets assume implement two separate 2 release lines 0.x.0 and v2 (skipping v1?) 

Some personal worries?
- We going to lose ability to migrate projects step by step (killing incentive to update)
- Models with the same name going to produce some odd behaviour andnerrors (rather import errors)
- We going to have v0.x.x and v2.0.0 releases happening simultanously 

Benefits?

- We do not need to choose new name
- Import size is smaller.
- ?

My take is to relase both (possibly under single name and version) and at some point deprecate v1 client and subsequently remove it. Versioning can stay 0.x.0 or go into v50.x.x - personally do not care as long we have both clients available under single package for migrators.

Reply:
> We going to lose ability to migrate projects step by step (killing incentive to update)

I thought that `go.mod` supports having multiple versions of the same module but a quick search online showed that is not possible indeed 😞 .  Thanks for the discussion and the clarifications!